### DISCUSSION TITLE
How add more tray icons?

### DISCUSSION THREAD

**Discussion by Ahplla** [Rem0o/FanControl.Releases#1046] [Q&A]
I have the CPU tray icon, but I would like to add the GPU temperature as well. I didn't find the option.

**Comment by Snacko** [comment]
Just trying out FC and I also cannot find where you can add a GPU Temp to the Systray..  Where is this setting hidden??

**Comment by Snacko** [comment]
Ok, I found it.  You have to go into the hamburger menu on the top/left and select 'Appearance'.  Then, on the left is the Systray Icons editor.
